mapie/classification.py

```diff
@@ -888,6 +888,63 @@ def _find_lambda_star(
             lambda_star = lambda_star[0]
         return lambda_star
 
+    def _get_classes_info(
+            self, estimator: ClassifierMixin, y: NDArray
+    ) -> Tuple[int, NDArray]:
+        """
+        Compute the number of classes and the classes values
+        according to either the pre-trained model or to the
+        values in y.
+
+        Parameters
+        ----------
+        estimator : ClassifierMixin
+            Estimator pre-fitted or not.
+        y : NDArray
+            Values to predict.
+
+        Returns
+        -------
+        Tuple[int, NDArray]
+            The number of unique classes and their unique
+            values.
+
+        Raises
+        ------
+        ValueError
+            If number of calibration labels is higher than number of labels
+            for training (in prefit setting)
+        Warning
+            If number of calibration labels is lower than number of labels
+            for training (in prefit setting)
+        """
+        n_unique_y_labels = len(np.unique(y))
```

Comment:
Do you take into account the use case when we do have the same number of labels but maybe in the training set you have [0, 1, 2] and in the calibration you have [1, 2, 3]? 

I'm not sure this is taken into account.

Reply:
Or tested for.

Reply:
Tested in `test_y_1_to_l_minus_1`

Reply:
and in `def test_same_result_y_numeric_and_1_to_l_minus_1`